libs/oeth/swap/src/components/ApyChart.tsx

```diff
@@ -212,74 +213,67 @@ export const ApyChart = (props: StackProps) => {
-        >
-          <Typography>
-            {intl.formatMessage({ defaultMessage: 'APY' })}
-          </Typography>
-          <Stack direction="row">
-            {limitOptions.map((d) => (
-              <Button
-                key={`duration-${d?.value ?? 'All'}`}
-                size="small"
-                onClick={() => {
-                  setLimit(d);
-                }}
-                sx={{
-                  minWidth: 0,
-                  width: 40,
-                  color: 'text.primary',
-                  ...(d.value === limit.value && {
-                    backgroundColor: (theme) =>
-                      theme.palette.background.default,
-                  }),
-                }}
-              >
-                {intl.formatMessage(d.label)}
-              </Button>
-            ))}
+    <Card>
+      <CardHeader
+        title={
+          <Stack
+            direction="row"
+            alignItems="center"
+            justifyContent="space-between"
+          >
+            <Typography>
+              {intl.formatMessage({ defaultMessage: 'APY' })}
+            </Typography>
+            <Stack
+              direction="row"
+              sx={{
+                borderRadius: 1,
+                padding: 0.25,
+                backgroundColor: 'grey.700',
+                gap: 0.5,
+              }}
+            >
+              {limitOptions.map((d) => (
+                <Button
+                  key={`duration-${d?.value ?? 'All'}`}
+                  variant="text"
+                  size="small"
+                  onClick={() => {
+                    setLimit(d);
+                  }}
+                  sx={{
+                    color: 'text.secondary',
+                    borderRadius: 1,
+                    minWidth: 0,
+                    width: 40,
+                    ...(d.value === limit.value && {
+                      background: (theme) =>
+                        theme.palette.background.gradientSelected,
+                    }),
+                  }}
+                >
+                  {intl.formatMessage(d.label)}
+                </Button>
+              ))}
+            </Stack>
           </Stack>
-        </Stack>
-      }
-    >
+        }
+      />
       <Stack>
         <Stack
           direction="row"
           alignItems="flex-start"
           justifyContent="space-between"
           sx={{
-            paddingInline: {
-              xs: 1.5,
-              md: 3,
-            },
+            paddingX: { xs: 2, md: 3 },
+            paddingTop: 1.5,
           }}
         >
           <Stack>
             {apiesLoading ? (
               <Skeleton width={60} height={24} />
             ) : (
-              <Typography
-                fontSize={24}
-                fontWeight={700}
-                fontFamily="Sailec"
-                color="primary.contrastText"
-              >
+              <Typography fontSize={24} fontWeight={700} fontFamily="Sailec">
```

Comment:
Is there a place for font families in the theme?

Reply:
the main ones are defined in `theme.typography.fontFamily` but it is not meant to be used "as-is". I have seen people adding other font families to the theme directly by adding custom props (i.e. `theme.typography.monospace`), but I think it's preferable to define text variants embedding the right font directly and just use font name for one-off customizations